AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsPrivate.h

```diff
@@ -38,12 +38,23 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @param eventName  event name.
  * @param properties dictionary of properties.
- * @param transmissionTarget Transmission target to associate with the event.
+ * @param transmissionTarget transmission target to associate with the event.
```

Comment:
@jaelim-ms FYI; i think you had a comment earlier that contradicted this feedback.

Reply:
Sorry but I can't recall it. 😄 
We have lots of places with inconsistency for this one. All descriptions for param, return and whatever in the doc should start with uppercase, we can make incremental changes or revisit and fix at once.
For this specific line, would be better not to change since this is not a related change of the PR.